### DISCUSSION TITLE
Is this a gui-manager only or it includes nginx?

### DISCUSSION THREAD

**Discussion by KeenMaron** [NginxProxyManager/nginx-proxy-manager#1621] [Q&A] [answered]
Hi, 

sorry for my newbish question. I already have set up my nginx on my raspberryPi. But i'm losing the overview of all that thousand of config files. 

I found this nice gui for nginx and was happy about a kind of visualisation what your nginx proxy is doing and how it is set up.

My question is:
Does the gui-manger work with the already installed nginx on my Pi? Or should i remove nginx and install only the docker image of the manager which includes nginx?

Thanks! :)

**Comment by chaptergy** [comment, marked as answer]
This docker container includes an nginx installation, so there should not be any other nginx installations on the same host using the same ports as the docker container with npm. It does not support using existing configurations, you need to set the proxies up using the gui.

**Reply by KeenMaron** [reply]
wow thanks for the fast answer!
